PublicMapping/districtbuilder-classic
# Fix geounit geometry unioning during template plan creation

## Overview

Template plan creation using the default Virginia data wasn't working due to the way it was trying to create MultiPolygons, and it was exceeding the memory availability in the VM to boot. This fixes both problems.

This also adds database view creation to the `load_development_data` script, which is necessary for Geoserver to work.

### Checklist


### Demo

After doing these two things, the map starts to look slightly less sad.

<img width="1552" alt="screen shot 2018-01-30 at 1 27 28 pm" src="https://user-images.githubusercontent.com/447977/35584183-3f643cf0-05c2-11e8-9bd2-0c43016ee0ad.png">


### Notes

- This makes the full `load_development_data` process take quite a bit longer because it's not erroring out on each district load. This is a good thing, but it eats into our time savings somewhat.
- I linted and removed `import *` statements from `tasks.py`; I don't think I broke anything but it's possible I did.
- There are still some errors during the Geoserver configuration step, of the form `[2018-01-30 18:20:24 +0000] [1111] [WARNING] Could not create "simple_dc_vtd" simple feature type`. It's not clear what, if anything, this is affecting.

## Testing Instructions

 * If you don't have non-simplified data already loaded, find a time when you have about 12 hours to spare and run `./scripts/load_development_data` using the non-simplified geometries.
 * If you do have non-simplified data loaded, start up a server using `./scripts/server` and then `docker compose exec vagrant_django_1 bash` inside the VM.
 * `./manage.py shell_plus` and then from Python `Plan.objects.all().delete()`
 * Load templates: `./manage.py setup config/config.xml -t` -- it should take a couple hours and not throw any errors.
 * Generate database views: `./manage.py setup config/config.xml --views` -- should be fairly quick, less than a minute.
 * Configure Geoserver: `./manage.py setup config/config.xml -G` -- should take a few minutes, may throw errors of the form above.
 * Log in as a guest, and confirm that every District Layer has an additional Plan, in addition to the "Blank" plan.
 * View one of the non-Blank plans and confirm that districts appear on the map.

Fixes #154372452
